addons/binding/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/json/response/ICloudAccountDataResponse.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.icloud.internal.json.response;
+
+import java.util.ArrayList;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Serializable class to parse the device information json response
+ * received from the Apple server.
+ *
+ * @author Patrik Gfeller - Initial Contribution
+ *
+ */
+public class ICloudAccountDataResponse {
+
+    @SerializedName("content")
+    private ArrayList<ICloudDeviceInformation> iCloudDeviceInformationList;
```

Comment:
Please prefer the interfaces of these classes over the actual implementations so, make this a `List<ICloudDeviceInformation>` instead of `ArrayList<ICloudDeviceInformation>`

Reply:
... done.